clang/unittests/Analysis/FlowSensitive/TransferTest.cpp

```diff
@@ -2313,6 +2313,42 @@ TEST(TransferTest, AssignmentOperatorWithInitAndInheritance) {
          ASTContext &ASTCtx) {});
 }
 
+TEST(TransferTest, CXXOperatorCallExprEqualReturnsVoid) {
```

Comment:
```suggestion
TEST(TransferTest, AssignmentOperatorReturnsVoid) {
```

(For consistency with tests above.)